requirements.txt

```diff
@@ -1,3 +1,4 @@
+urllib3
```

Comment:
We never run requirements.txt in our framework, but I think now it's time to use, please update the file as:
```
attrdict==2.0.1
paramiko==2.5.0
pytest==5.4.3
pytest-services==1.3.1
pytest-mock==1.10.4
pytest-xdist==1.34.0
requests==2.22.0
unittest2==1.1.0
urllib3==1.25.3
PyYAML==5.1.2
pexpect==4.7.0
```

Reply:
I will commit another PR for virt-who job to run the requirements.txt